lib/routes/anthropic/news.ts

```diff
@@ -61,7 +62,7 @@ async function handler() {
                     }
                 });
 
-                item.description = content.html();
+                item.description = content.html() || undefined;
```

Comment:
.html() will [return null](https://github.com/cheeriojs/cheerio/blob/6364a47674088c416bcd64afcf8b8b2f75d586fb/src/api/manipulation.ts#L966)
```suggestion
                item.description = content.html();
```